public/app/core/logs_model.ts

```diff
@@ -0,0 +1,29 @@
+export enum LogLevel {
+  crit = 'crit',
+  warn = 'warn',
+  err = 'error',
+  error = 'error',
+  info = 'info',
+  debug = 'debug',
+  trace = 'trace',
+}
+
+export interface LogSearchMatch {
```

Comment:
thinking of having these pure types definitions in app/types, not sure if that is a good idea or not.

Reply:
Yeah, still *exploring* this.